### DISCUSSION TITLE
Using the Confirmation URL when attempting to bypass Email Client tracking or prefetching

### DISCUSSION THREAD

**Discussion by jameswoodley** [supabase/supabase#28623] [Questions]
My users are having trouble with magic links if they use something like Office 365.. I think either the links are pre-fetched for some reason, or they are broken with tracking links. 

I've followed the instructions in: [https://supabase.com/docs/guides/auth/auth-email-templates#limitations]() and changed my Magic Link email template to `<a href="{{ .SiteURL }}/auth/magic-link?confirmation_url={{ .ConfirmationURL }}">Click Here</a>`

The confirmation url is `http://127.0.0.1:54321/auth/v1/verify?token=pkce_1615cf11f2fa8873d181b3e6aa6b5b776792f701c50f429fcca52193&type=magiclink&redirect_to=https://localhost:5173` with the auth/v1/verify being Supabase and not my product. Normally I send them to code I've created that I can get the token etc from the querystring and call `verifyOtp`, but I can't do that with the verify URL. 

How would one log someone in using the Confirmation Url?

**Comment by grantbrewster** [comment]
I've been having this same issue. I ended up having to use OTP for everything which I don't love. 

I'm really curious how other companies and auth providers get around email clients automatically clicking links and expiring them. Is this uniquely a supabase auth problem?
